Main/Home.css

```diff
@@ -0,0 +1,90 @@
+main{
+    display: grid;
+    grid-template-columns:1fr;
+    grid-template-areas: 
+     "header"
+     "hr"
+     "main-content"
+    ;
+}
+header{
+    width: 70%;
+    margin: 0 auto;
+    grid-area: header;
+}
+.navigation{
+    display: flex;
+    align-items:baseline;
+    justify-content: space-between;
+}
+.logo{
+    font-size: 5rem;
+    color:var(--color-neon-blue);     
+    flex-grow: 1;
+}
+.img-logo{
+    height: 5rem;
+}     
+.secondary-link{
+    display: flex;
+    margin: 0rem 2rem;
+}
+.secondary-logo{
+    color:rgba(107,124,147,0.7);
+    font-size: 2.5rem;
+    margin: 1rem;
+}
+.secondary-logo:hover{
+    color: var(--color-grey);
+}
+hr{
+    grid-area: hr;

```

Comment:
why putting hr tag in grid area?

Reply:
hr tag I have not put in header bacuse it is changing according to the header width grid area because i want full width if i will not it goes below the page

Reply:
okay